packages/amplify_core/lib/src/category/amplify_auth_category.dart

```diff
@@ -95,6 +256,186 @@ class AuthCategory extends AmplifyCategory<AuthPluginInterface> {
   ///
   /// For Cognito flows, including a [password] will initiate an SRP sign-in,
   /// while excluding the [password] will initiate a custom auth sign-in.
+  ///
+  /// <?code-excerpt "doc/lib/auth.dart" region="imports"?>
+  /// ```dart
+  /// import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+  /// import 'package:amplify_flutter/amplify_flutter.dart';
```

Comment:
Should there be a blank line between the imports and the first fn? It just looks a little crowded when viewing it in an IDE.

![Screen Shot 2023-04-12 at 10 35 56 AM](https://user-images.githubusercontent.com/20613561/231491692-243a805e-fd2b-48c6-8f18-e5ebd5e8004a.png)

Reply:
The spacing issue is weird - I played around with it and it's not so straightforward to solve unfortunately. I'll do a bit more digging if I have time but at the moment, this is currently the best I can do.